CONTRIBUTING.md

```diff
@@ -333,8 +333,7 @@ and in the interest of fostering an open and welcoming community, we pledge to
 respect all people who contribute through reporting issues, posting feature
 requests, updating documentation, submitting pull requests or patches, and other
 activities to any of the projects under the containers umbrella. The full code of conduct guidelines can be
-include Buildah, Conmon, Image, Podman, Ramalama, Storage, Skopeo, et al.
-found [here]([https://github.com/containers](https://github.com/caradelia/ramalama/blob/main/CODE-OF-CONDUCT.md)).
+found [here](https://github.com/caradelia/ramalama/blob/main/CODE-OF-CONDUCT.md).
```

Comment:
I am not sure if this is correct, it's pointing to your own fork. Should be containers/common?

Reply:
Yes, correct,I need to fix.